core/conversion/evaluators/aten.cpp

```diff
@@ -464,7 +464,38 @@ auto aten_registrations TRTORCH_UNUSED =
                       LOG_WARNING("Warning from TorchScript: " << *warning);
                       return {};
                     },
-                    EvalOptions()});
+                    EvalOptions()})
+        .evaluator({c10::Symbol::fromQualString("aten::is_floating_point"),
+                    [](const torch::jit::Node* n, kwargs& args) -> c10::optional<torch::jit::IValue> {
+                      auto tensor_var = args.at(n->input(0));
+                      if (tensor_var.isITensor()) {
+                        auto tensor = tensor_var.ITensor();
+                        auto t = tensor->getType();
+                        return (t == nvinfer1::DataType::kFLOAT || t == nvinfer1::DataType::kHALF);
+                      } else {
+                        auto tensor = tensor_var.unwrapToTensor();
+                        auto t = tensor.scalar_type();
+                        return at::isFloatingType(t);
+                      }
+                    },
+                    EvalOptions().validSchemas({
+                        "aten::is_floating_point(Tensor self) -> (bool)",
+                    })})
+        .evaluator({c10::Symbol::fromQualString("aten::Int"),
```

Comment:
Is there a better way to handle this? Type casts dont have meaning in TensorRT. Are what are the possible use cases of this op and how do we handle them?

Reply:
Tagging @inocsin for viz. I have similar question as well. Can anyone let me know the usage of this op ? 
In line 487, there is a check if node input is an ITensor or not. ITensor() true case happens when  node input is in ctx->value_tensor_map right ? What is the layer preceeding this layer usually ? Can you add test case if there are no blockers ?

Reply:
```
  %2270 : bool = aten::is_floating_point(%input174.1) # <string>:3:6
  %2271 : bool = prim::If(%2270) # <string>:3:6
```
It was used as in If-Else

Reply:
aten::Int, aten::is_floating_point test case is here https://github.com/NVIDIA/TRTorch/pull/296 , the evaluator test framework was not ready at that time, so I submit anothor commit with test case.